stripe/_invoice.py

```diff
@@ -5620,7 +5620,6 @@ class VoidInvoiceParams(RequestOptions):
     The status of the invoice, one of `draft`, `open`, `paid`, `uncollectible`, or `void`. [Learn more](https://stripe.com/docs/billing/invoices/workflow#workflow-overview)
     """
     status_transitions: StatusTransitions
-    subscription: Optional[ExpandableField["Subscription"]]
```

Comment:
is this intentional?

Reply:
This should have gone in as part of prior PRs - not sure why it got left out - yes this is intentional - it is one of the overrides we added